regression-tests/CMakeLists.txt

```diff
@@ -8,35 +8,43 @@ find_package(cppfront REQUIRED)
 
 set(REGRESSION_TESTS_DIR "${CMAKE_CURRENT_LIST_DIR}/../cppfront/regression-tests")
 
+include("CppfrontDeveloper.cmake")
```

Comment:
Since this is unconditionally included in exactly this one place, is it worth factoring out to a different file? I'd prefer it were all inlined if there's no real leverage to be gained by splitting it up.

I would also suggest creating the `cppfront_update_test_results` target up front and using `add_dependencies` in `cppfront_dev_setup_to_update_test_results` (which could actually be inlined, too). This avoids both the use of `cmake_language(DEFER)` and a custom directory property.

Reply:
Thank you.

I'll try that
as I actually use it
to test that the `fix` commits above continue to work.

Maybe I should add tests for this CMake project itself.

Reply:
Applied.